pages/desktop/details.py

```diff
@@ -565,3 +599,105 @@ def ratings_card_summary(self):
         @property
         def rating_login_button(self):
             return self.find_element(*self._login_to_rate_button_locator)
+
+        @property
+        def rating_stars(self):
+            # waits for the ratings stars to be fully loaded and editable
+            self.wait.until(
+                expected.invisibility_of_element_located(
+                    self._loaded_rating_stars_locator
+                )
+            )
+            return self.find_elements(*self._rating_stars_locator)
+
+        @property
+        def selected_star_highlight(self):
+            # waits for the ratings stars to be fully loaded and editable
+            self.wait.until(
+                expected.invisibility_of_element_located(
+                    self._loaded_rating_stars_locator
+                )
+            )
+            return self.find_elements(*self._highlighted_star_locator)
+
+        @property
+        def delete_rating_link(self):
+            # waits for the ratings stars to be fully loaded and editable
+            self.wait.until(
+                expected.invisibility_of_element_located(
+                    self._loaded_rating_stars_locator
+                )
+            )
+            return self.find_element(*self._delete_rating_link_locator)
+
+        def delete_confirm_button(self):
```

Comment:
maybe this should have click in the name. This seems like it is named like a property not a function.
```suggestion
        def click_delete_confirm_button(self):
```